File: java/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/DummyActivity.java
package com.pubscale.sdkone.example.abtestexample;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.pubscale.sdkone.example.abtestexample.databinding.ActivityDummyBinding;

public class DummyActivity extends AppCompatActivity {

    private ActivityDummyBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityDummyBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.homeButton.setOnClickListener(view -> {
            onBackPressed();
        });
    }
}

File: kotlin/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/DummyActivity.kt
package com.pubscale.sdkone.example.abtestexample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.pubscale.sdkone.example.abtestexample.databinding.ActivityDummyBinding

class DummyActivity : AppCompatActivity() {

    private lateinit var binding: ActivityDummyBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityDummyBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.homeButton.setOnClickListener { onBackPressed() }
    }
}